cobra/util/solver.py

```diff
@@ -20,7 +20,7 @@
 import optlang
 import sympy
 
-from cobra.exceptions import OptimizationError
+from cobra.exceptions import OptimizationError, _OPTLANG_TO_EXCEPTIONS_DICT
```

Comment:
I'd feel better if `_OPTLANG_TO_EXCEPTIONS_DICT` was not marked as private or exposed to public in some other way.